From a0e1001027125cf25aabdd023f74488f44ebc906 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@gmail.com>
Date: Wed, 27 Mar 2019 22:27:33 +0200
Subject: [PATCH] CMake: Fix installation of man pages.

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e56cd44e..5eff8858 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -647,7 +647,7 @@ install (TARGETS ${SNDFILE_STATIC_TARGET} ${SNDFILE_SHARED_TARGET} ${sdnfile_PRO
 
 endif (ENABLE_PACKAGE_CONFIG)
 
-if (Unix)
+if (UNIX)
 	set (man_MANS
 		man/sndfile-info.1
 		man/sndfile-play.1
@@ -659,7 +659,7 @@ if (Unix)
 		man/sndfile-interleave.1
 		#man/sndfile-deinterleave.1
 		man/sndfile-salvage.1)
-	install (FILES ${man_MANS} DESTINATION ${CMAKE_INSTALL_MANDIR}/man1 PATTERN "*.1")
+	install (FILES ${man_MANS} DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
 endif ()
 
 if (ENABLE_BOW_DOCS)
-- 
2.17.1

